src/components/Hero/Hero.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import shieldjsLogo from '@/assets/shieldjs.svg'
+import {
+  HeroSection,
+  HeroDescription,
+  HeroSubtitle,
+  HeroTitle,
+  HeroContent,
+  HeroImage,
+  HeroButton
+} from './Hero.styles'
+
+export const Hero: React.FC = () => {
+  return (
+    <HeroSection>
+      <HeroContent>
+        <HeroTitle>Shield.js</HeroTitle>
+        <HeroSubtitle>The new frontend project by One&nbsp;Beyond</HeroSubtitle>
```

Comment:
should we use i18n for whole page? or just a section like an example?

Reply:
I would put the texts in the i18n as we have in the pages that already exist

Reply:
I'll do it